rotkehlchen/tests/unit/decoders/test_compound_v3.py

```diff
@@ -261,3 +260,247 @@ def test_compound_v3_repay(database, ethereum_inquirer, ethereum_accounts):
+    gas_fees, supply_amount, position_amount = '0.000011015992753122', '0.1', '0.099999'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=base_accounts[0],
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_BASE_USDC,
+            balance=Balance(amount=FVal(supply_amount)),
+            location_label=base_accounts[0],
+            notes=f'Deposit {supply_amount} USDC to compound v3',
+            counterparty=CPT_COMPOUND_V3,
+            address=string_to_evm_address('0xb125E6687d4313864e53df431d5425969c15Eb2F'),
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+            asset=Asset('eip155:8453/erc20:0xb125E6687d4313864e53df431d5425969c15Eb2F'),  # cUSDCv3
+            balance=Balance(amount=FVal(position_amount)),
+            location_label=base_accounts[0],
+            notes=f'Receive {position_amount} cUSDCv3 from compound v3',
+            counterparty=CPT_COMPOUND_V3,
+            address=ZERO_ADDRESS,
+        )]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('scroll_accounts', [['0xe53Ca0bB4E6F22514Aa1c1ABFd9634d52A808546']])
+def test_scroll_withdraw(database, scroll_inquirer, scroll_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x0ec09157328bba439e276c83a2c3364c390d430b5bc0bddf3169d475b6fbfdee')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=scroll_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1712739257000)
+    gas_fees, withdraw_amount = '0.000105346', '5001.003801'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.SCROLL,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=scroll_accounts[0],
+            notes=f'Burned {gas_fees} ETH for gas',  # TODO: Total gas fees should be included here
```

Comment:
Hmm? What is this todo?

Reply:
it failed before, but probably not because of this. I must have missed something... removing it now.